release.yaml

```diff
@@ -1,4 +1,6 @@
 longhorn:
   - 101.2.0+up1.4.0
+  - 102.2.0+up1.4.1
 longhorn-crd:
   - 101.2.0+up1.4.0
+  - 102.2.0+up1.4.1
```

Comment:
This should only contain the charts that are being released

Reply:
Resolved.